app/locales/sk.json

```diff
@@ -0,0 +1,141 @@
+{
+  "Postal code": "PSČ",
+  "Important": "Doležité",
+  "Page not found": "Stránka nenájdená",
+  "We can't find the page or service you're looking for": "Stránka alebo služba žial nebola nájdená",
+  "The link you clicked may be broken, or the page no longer exists": "Odkaz, na ktorý ste klikli, je nefunkčný alebo stránka už neexistuje",
+  "Something went wrong!": "Niekde nastala chyba!",
+  "Something must have gone wrong": "Niekde musela nastať chyba",
+  "Send an e-mail to": "Pošli email na",
+  " if you can't get it to work.": " ak sa Vám to nedarí sprevádzkovať.",
+  "Before proceeding please note down the following link": "Pred pokračovaním si prosím uložte nasledujúci link",
+  "Save the following link somewhere, or bookmark it in your browser.": "Uložte si tento odkaz alebo si ho pridajte do záložiek.",
+  "That way you can update your report if your health condition changes.": "Touto cestou viete aktualizovať Váš report v prípade zmeny zdravotného stavu.",
+  "I've noted down the link.": "Zapísal som si tento odkaz.",
+  "Proceed": "Pokračovať",
+  "The link has been copied": "Odkaz bol skopírovaný.",
+  "Could not copy the link, please copy it yourself": "Odkaz sa nedá skopírovať, skopírujte si ho manuálne",
+  "Contributors": "Prispievatelia",
+  "This website is made with help from volunteers. In no particular order:": "Táto stránka je vytvorená s pomocou dobrovoľných prispievateľov. Bez konkrétneho poradia:"
```

Comment:
Missing comma here

Reply:
thank you, will push in a batch fixed together with others